# Messaggi del classificatore

Header header

uint8 SYNC_FAILED = 6
uint8 type

# If SYNC_FAILED
uint8 TOO_HARD = 1
uint8 sync_result